### PR TITLE
Update code.json

### PR DESCRIPTION
**Pull request by SAMZONG**
more translate to Chinese

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

- Documentation
- i18n code transalte to Chinese of almost message field

**Special notes for your reviewer**:

- make karmada is great

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

**Comment by pigletfly** [comment]
> /assign @pigletfly Please get your student involved.

ok

File: i18n/zh/code.json [hunk 1]
```diff
@@ -124,74 +124,74 @@
     "description": "The label used to tell the user that he's browsing an unmaintained doc version"
   },
   "theme.docs.versions.latestVersionSuggestionLabel": {
-    "message": "For latest stable documentation, see the {latestVersionLink} ({versionLabel}).",
+    "message": "查看当前最新版本文档, 点击 {latestVersionLink} ({versionLabel}).",
     "description": "The label userd to tell the user that he's browsing an unmaintained doc version"
```
Comment (windsonsea) [inline i18n/zh/code.json:128]:
```suggestion
    "description": "The label used to tell the user that he's browsing an unmaintained doc version"
```

File: i18n/zh/code.json [hunk 2]
```diff
@@ -124,74 +124,74 @@
...
   },
   "theme.docs.versions.latestVersionLinkLabel": {
-    "message": "最近稳定版本",
+    "message": "最新稳定版本",
     "description": "The label used for the latest version suggestion link label"
```
Comment (windsonsea) [inline i18n/zh/code.json:132]:
```suggestion
    "description": "The link label used for the latest version suggestion"
```

**Review by windsonsea — COMMENTED** [review]
pick nits 😃

**Comment by RainbowMango** [comment]
@SAMZONG You might have to [squash your commits](https://github.com/karmada-io/karmada/blob/master/docs/contributors/guide/github-workflow.md#squash-commits).

**Comment by SAMZONG** [comment]
@RainbowMango it's done.

**Review by RainbowMango — COMMENTED** [review]
/lgtm
/approve

Thanks.

**Review by rgrupesh — COMMENTED** [review]
/lgtm

**Comment by RainbowMango** [comment]
@SAMZONG You might need to sign your commits.
Here is a guideline:https://github.com/karmada-io/website/pull/89/checks?check_run_id=7065413929

**Comment by SAMZONG** [comment]
@RainbowMango Thanks, I got it, pls confirm

**Review by RainbowMango — COMMENTED** [review]
/lgtm

**Pull request merged**